Data-Structures/src/main/java/graph/Graph.java

```diff
@@ -67,4 +67,38 @@ public Graph() {
     public int size() {
         return this.nodes.size();
     }
+
+    // accept a starting node and perform a breadth-first traversal on the graph
+    // return a collection of the nodes in the order they were visited
+    public String breadthFirstTraversal(Node start) {

```

Comment:
You can make the parameter a generic, like `Node<T> start` to prevent the "Unchecked assignment" warnings IntelliJ is giving you.